### DISCUSSION TITLE
initialization retry failed fan control

### DISCUSSION THREAD

**Discussion by Cah0s** [Rem0o/FanControl.Releases#2858] [Q&A]
I just see the tool in a Video on Youtube where the dude says is a good tool... i don't think so, same other tools, no function.

I only open the tool(Downloaded from here the actually version c209, first start just give me back instantly "initialization retry failed" nothing more, no code no any more info, nothing just retry failed... yeah cool. i tried find some in google but notginh what help and i searched here, nothing what help. After this first start, when i reopen the tool no show nothing more, the failed info just on first time.

I need to do a curse or se any professional crazy Tutorial or what the problem with the app ? i Mean initialization retry failed ... niceeee i will find now, exactly now the problem with the error info.... wait, there no info, just failed. tnx, cool tool.

When i nerver used any fan tool, why this don't can function right ? why is no start setup what check all and give back for some who he no can detect ????? What i do now with the info "initialization retry failed" ?????

Is a New Laptop, nothing changed, no one Tool like this, nothing. is no possible that for all functions but for me, again, no. Idk but all type of Tool for Fan's are bugged and cringe, no possible.

I hope someone have a answer, one what helps... like Download and try again: i already tried. Restart Laptop, yeaaah already done, oh oh wait, check the configs... yes babe, its already done too. Any more ?

**Comment by Rem0o** [comment]
Laptops are not supported by FanControl.  Laptops are not meant for third part fan control.
